modules/enabling-additional-enabled-capabilities.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// *post_installation_configuration/cluster-capabilities.adoc
+
+[id="setting_additional_enabled_capabilities_{context}"]
+= Enabling the cluster capabilities by setting additional enabled capabilities
+
+As a cluster administrator, you can enable the cluster capabilities by setting `additionalEnabledCapabilities`.
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (oc).
+* You have installed the link:https://stedolan.github.io/jq/[jq package].
+
+.Procedure
+
+. View the additional enabled capabilities by running the following command:
++
+[source,terminal]
+----
+$ oc get -o json clusterversion version | jq -c '.spec.capabilities.additionalEnabledCapabilities'
+----
+
++
+.Example output
+[source,terminal]
+----
+["openshift-samples"]
+----
+
+. To set the `additionalEnabledCapabilities` run the following command:
++
+[source,terminal]
+----
+$ oc patch clusterversion/version --type merge -p '{"spec":{"capabilities":{"additionalEnabledCapabilities":["openshift-samples", "marketplace"]}}}'
+----
+
++
+[IMPORTANT]
+====
+It is not possible to disable a capability which is already enabled in a cluster. The cluster version Operator (CVO) continues to reconcile the capability which is already enabled in the cluster.
+====
+
+
```

Comment:
I couldn't tell if these were meant to be indented. Ignore if I'm wrong 😄

Reply:
done. Thanks